backend/hearing/src/main/java/org/pucar/dristi/enrichment/HearingRegistrationEnrichment.java

```diff
@@ -53,7 +53,12 @@ public void enrichHearingRegistration(HearingRequest hearingRequest) {
             String idFormat = configuration.getHearingFormat();
 
             List<String> hearingIdList = idgenUtil.getIdList(hearingRequest.getRequestInfo(), tenantId, idName, idFormat, 1, false);
-            hearing.setHearingId(hearingIdList.get(0));
+            hearing.setHearingId(hearing.getCnrNumbers().get(0) +"-"+hearingIdList.get(0));
+
+            if(null != hearing.getCourtCaseNumber())
+                hearing.setCaseReferenceNumber(hearing.getCourtCaseNumber());
+            else
```

Comment:
@nileshgupta111  courtCaseNumber is to be renamed to ccstNumber. See this ticket -- https://github.com/pucardotorg/dristi/issues/1686. 

Coordinate with @manimaarans  on when this is being done.